spec/compiler/macro/macro_methods_spec.cr

```diff
@@ -10,6 +10,37 @@ private def declare_class_var(container : ClassVarContainer, name, var_type : Ty
   container.class_vars[name] = var
 end
 
+private def exit_code_command(code)
+  {% if flag?(:win32) %}
+    %(cmd.exe /c "exit #{code}")
+  {% else %}
+    case code
+    when 0
+      "true"
+    when 1
+      "false"
+    else
```

Comment:
Why this distinction?

Reply:
Not sure. I just copied this helper 1:1 from `process_spec`.

Reply:
I don't think this is really necessary, but I also didn't want to touch that existing code. We might want to refactor it anyway (cf #12775).